index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>HEllo microverse</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
- [Optional] Please the text is `Hello Microverse!`, try paying attention to details it will be beneficial in future projects. 

Please do not merge a PR until it is approved by a code reviewer. Happy coding :+1: